spec/features/section_home_page.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'home page', type: :feature do
+
+  Warden.test_mode!
+
+  let!(:root_node) { Fabricate(:root_node) }
```

Comment:
I think I've fixed the fabricators so you don't need to create the root node explicitly anymore.